### ISSUE TITLE
Mouse Right Click makes the player not stop previous held button

### ISSUE THREAD

**Issue by Washiii** [tsoding/raycasting#28]
Hello, while testing the game I found an issue where while holding any movement key (W, A, S or D) and pressing Right Click (MB3) makes the player not able to stop the action he was performing.

If done multiple times for the same key it can add some speed up, I wasn't able to go to fast on the example, neither testing, but I think it is possible to get unlimited speed with this bug, therefore abusing some gameplay aspect in the future.

You can stop the bug yourself by doing the opposite action again an equal amount of times you did, for example, doing 3 times to the left (A) then 3 times to the right (D) would stop the bug and the player would comeback to normal.

https://github.com/tsoding/raycasting/assets/52114739/22b24a77-79e2-43f7-b1bc-ac7c041d5728

**I miss abusing this on flash games**

**Comment by Washiii** [comment]
I just found out a few things about this bug, pressing TAB also works.

And to stop the movement you just the the "release" event for the key, so you don't need to counter it with opposite commands. You also can't stack it sadly.

**Comment by rexim** [comment]
Yeah, the control handling is pretty rudimentary, we will fix it later.

**Comment by y4my4my4m** [comment]
This is duplicate of #36 actually
